.cargo/audit.toml

```diff
@@ -0,0 +1,10 @@
+[advisories]
+ignore = [
+    # RSA Marvin Attack in `rsa`, dragged in through rustcrypto
+    # this is only used in dev builds
+    "RUSTSEC-2023-0071",
```

Comment:
rsa is also brought in through adb_client. do you think we should care about that or mention it in the comments here?

Reply:
@untitaker see above

Reply:
did some spelunking and i think i figured adb-client isn't exploitable either (but even if, i'm not sure there is a realistic attack here that doesn't require physical hw access by the attacker...)